packages/react/src/components/Text/Text.tsx

```diff
@@ -22,7 +22,7 @@ export type TextProps<T extends React.ElementType> = PolymorphicProps<
 function Text<T extends React.ElementType>({
   as,
   children,
-  dir,
+  dir = 'auto',
```

Comment:
Are we sure this is what this should be defaulted to? For some reason I thought we specifically didn't add a default, but I can't remember the reasoning.

Reply:
This is reverting to what the default was, I (mistakenly) removed it [here](https://github.com/carbon-design-system/carbon/pull/14580/files#diff-ffaea91692b65e84832ac2f9545028d2f6944c8fe3a8120d1ccd6fed473a9b7fL25-L25)